android/src/main/java/com/urbanairship/reactnative/ReactMessageViewManager.java

```diff
@@ -0,0 +1,57 @@
+/* Copyright Airship and Contributors */
+
+package com.urbanairship.reactnative;
+
+import androidx.annotation.Nullable;
+
+import com.facebook.react.common.MapBuilder;
+import com.facebook.react.uimanager.SimpleViewManager;
+import com.facebook.react.uimanager.ThemedReactContext;
+import com.facebook.react.uimanager.annotations.ReactProp;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import javax.annotation.Nonnull;
+
+public class ReactMessageViewManager extends SimpleViewManager<ReactMessageView> {
+
+    public static final String REACT_CLASS = "UARCTMessageView";
```

Comment:
Is this correct?

Reply:
I only ask because I notice it's a class name in the iOS native code

Reply:
This is just the glue from the JS side to the view manager. I reused the iOS class name so we could have a single .js file

Reply:
It connects this line `var UARCTMessageView = requireNativeComponent('UARCTMessageView', UAMessageView);
`